doc/modules/computing.rst

```diff
@@ -504,20 +504,47 @@ Links
   - `Scipy sparse matrix formats documentation <https://docs.scipy.org/doc/scipy/reference/sparse.html>`_
 
 Parallelism, resource management, and configuration
-=====================================================
+===================================================
 
 .. _parallelism:
 
-Parallel and distributed computing
------------------------------------
+Parallelism
+-----------
 
-Scikit-learn uses the `joblib <https://joblib.readthedocs.io/en/latest/>`__
-library to enable parallel computing inside its estimators. See the
-joblib documentation for the switches to control parallel computing.
+Scikit-learn supports parallel implementations in two ways: via the `joblib
+<https://joblib.readthedocs.io/en/latest/>`_ library, or via OpenMP (through
+Cython, which allows to release the GIL).
+
+When the underlying implementation uses joblib, the number of jobs that are
+spawned in parallel can be controled via the ``n_jobs`` parameter. In most
+cases, scikit-learn uses joblib to rely on multi-processing rather than on
+multi-threading.
+
+OpenMP is used to parallelize low-level code written in Cython, relying on
```

Comment:
we don't use openMP is C code do we??

Reply:
Well OpenBLAS can use OpenMP and it's not Cython code.

Otherwise not yet, but there is nothing preventing us to : https://github.com/scikit-learn/scikit-learn/issues/6245

Might be better to stay general here, otherwise it will likely get outdated soon.